backendless_chef.yaml

```diff
@@ -578,11 +578,9 @@ Resources:
         Version: "2012-10-17"
         Statement:
           - Effect: "Allow"
-            Principal: "*"
+            Principal:
+              AWS: !GetAtt ChefRole.Arn
```

Comment:
the second one might have to be `!Sub arn:aws:iam::${AWS::AccountId}:role/${ChefServerIamRole}`